core/src/apps/common/request_passphrase.py

```diff
@@ -44,7 +44,7 @@
     await ctx.call(req, ButtonAck)
 
     text = Text("Enter passphrase", ui.ICON_CONFIG)
-    text.normal("Where to enter your", "passphrase?")
+    text.normal("Where do you want to", "enter your passphrase?")
```

Comment:
Before and after:
<img src="https://user-images.githubusercontent.com/42678794/65315264-11b38380-db98-11e9-806e-62a0c97e8a6d.png" width=300><img src="https://user-images.githubusercontent.com/42678794/65315284-15dfa100-db98-11e9-8e81-d06d14cbacab.png" width=300>